src/main/kotlin/no/nav/syfo/client/oppfolgingstilfelle/Oppfolgingstilfelle.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.syfo.client.oppfolgingstilfelle
+
+import java.time.LocalDate
+
+const val ARBEIDSGIVERPERIODE_DAYS = 16L
+
+data class Oppfolgingstilfelle(
+    val start: LocalDate,
+    val end: LocalDate,
+)
+
+fun Oppfolgingstilfelle.isActive(): Boolean = !isInactive()
+
+fun Oppfolgingstilfelle.isInactive(): Boolean =
+    LocalDate.now().isAfter(this.end.plusDays(ARBEIDSGIVERPERIODE_DAYS))
```

Comment:
Hvorfor tenker du det er bedre med extension functions i stedet for member functions her? 😊

Reply:
Har ikke tenkt så mye på det her siden koden er kopiert fra isdialogmote 😉

Vet ikke riktig om jeg har noen sterk oppfatning, men vi har vel ofte brukt extension functions for data classes (og ikke member function).